Source/WebCore/platform/network/CredentialStorage.cpp

```diff
@@ -187,4 +187,11 @@ void CredentialStorage::clearCredentials()
     m_pathToDefaultProtectionSpaceMap.clear();
 }
 
+#if USE(SOUP) || USE(CURL)
+Credential CredentialStorage::getFromPersistentStorage(const ProtectionSpace&)
+{
+    return Credential();
+}
+#endif
+
```

Comment:
I think we should change CredentialStorage.h.

~~~
#if PLATFORM(COCOA)
    // OS credential storage.
    WEBCORE_EXPORT static Credential getFromPersistentStorage(const ProtectionSpace&);
#endif
~~~